contrib/aws-cmake/CMakeLists.txt

```diff
@@ -336,11 +303,6 @@ list(APPEND AWS_PUBLIC_INCLUDES "${AWS_CRT_DIR}/include/")
 
 
 # aws-c-mqtt
-file(GLOB AWS_MQTT_SRC
```

Comment:
Guess we can remove

```cmake
SET(AWS_MQTT_DIR "${ClickHouse_SOURCE_DIR}/contrib/aws-c-mqtt")
```

from l. 72.

Also, do we still need submodule `aws-c-mqtt`?